db/migrate/20090619190332_create_sites.rb

```diff
@@ -1,4 +1,4 @@
-class CreateSites < ActiveRecord::Migration
+class CreateSites < ActiveRecord::Migration[5.1]
```

Comment:
As of Rails 5.0+ you are required to specify the version on `ActiveRecord::Migration`.  Else `rails db:migrate` will error out.

So added this to last know version of previous migrations (which was 5.1)